pontoon/machinery/views.py

```diff
@@ -113,6 +117,32 @@ def machine_translation(request):
         return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
 
 
+def machine_translation_caighdean(request):
+    """Get translation from Caighdean machine translation service."""
+    try:
+        path = request.GET["path"]
+        key = request.GET["key"]
+        locale_code = request.GET['locale']
+    except MultiValueDictKeyError as e:
+        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
+    try:
+        entity = Entity.objects.get(
+            key=key,
+            resource__path=path)
+    except Entity.DoesNotExist as e:
+        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
+    try:
+        text = entity.translation_set.get(locale__code="gd").string
+    except Translation.DoesNotExist:
+        return JsonResponse({})
+    try:
+        return JsonResponse(
+            dict(locale=locale_code,
+                 translation=Translator().translate(text)))
```

Comment:
You will make Adrian very happy if you format this like that:
```
return JsonResponse(
    dict(
        locale=locale_code,
        translation=Translator().translate(text)
    )
)
```

:-)

I don't have a strong opinion on this, but I'd use `{}` instead of `dict()`.